modules/viewing-metrics-for-the-single-model-serving-platform.adoc

```diff
@@ -27,11 +27,13 @@ endif::[]
 . Switch to the *Developer* perspective.
 . In the left menu, click *Observe*.
 ifdef::upstream,self-managed[]
-. As described in link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-latest-version}/html/building_applications/odc-monitoring-project-and-application-metrics-using-developer-perspective#odc-monitoring-your-project-metrics_monitoring-project-and-application-metrics-using-developer-perspective[monitoring project metrics^], use the web console to run queries for `caikit_*`, `tgi_*`, `ovms_*` or `istio_*` metrics.
+. As described in link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-latest-version}/html/building_applications/odc-monitoring-project-and-application-metrics-using-developer-perspective#odc-monitoring-your-project-metrics_monitoring-project-and-application-metrics-using-developer-perspective[monitoring project metrics^], use the web console to run queries for `caikit_*`, `tgi_*` and `ovms_*` serving runtime metrics. You can also run queries for `istio_*` metrics related to OpenShift Service Mesh.
 endif::[]
 ifdef::cloud-service[]
-. As described in link:https://docs.openshift.com/dedicated/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Dedicated) or link:https://docs.openshift.com/rosa/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Service on AWS), use the web console to run queries for `caikit_*`, `tgi_*`, `ovms_*` or `istio_*` metrics.
+. As described in link:https://docs.openshift.com/dedicated/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Dedicated) or link:https://docs.openshift.com/rosa/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Service on AWS), use the web console to run queries for `caikit_*`, `tgi_*` and `ovms_*` serving runtime metrics. You can also run queries for `istio_*` metrics related to OpenShift Service Mesh.
 endif::[]
 
-// [role="_additional-resources"]
-// .Additional resources
+[role="_additional-resources"]
+.Additional resources
+* link:https://docs.openvino.ai/2024/ovms_docs_metrics.html#available-metrics-families[OVMS metrics]
+* link:https://github.com/IBM/text-generation-inference?tab=readme-ov-file#metrics[TGIS metrics]
```

Comment:
Include caret symbols so that web browsers open these external links in new tabs. That is:

```
* link:https://docs.openvino.ai/2024/ovms_docs_metrics.html#available-metrics-families[OVMS metrics^]
* link:https://github.com/IBM/text-generation-inference?tab=readme-ov-file#metrics[TGIS metrics^]
```